sirepo/job_supervisor.py

```diff
@@ -293,6 +302,7 @@ def _purge_sim(db_file):
             d.status = job.FREE_USER_PURGED
```

Comment:
`JOB_RUN_PURGED`.  We don't care why, just that it was purged.

Reply:
Done. But why don't we care? In the future we may purge for other reasons (purge premium after n days or move to an archive after y days). I guess this could be a YAGNI thing but it seems like upside with no downside (the abstraction is exactly the same, just a phrase). More information in the db would be better than less. My general rule of thumb for writing code that is in line with your style is that when we don't really know how something is going to be used more info is better than less. Is that misguided and/or not true in this case?

Reply:
I understand that it may seem strange, but offer too specific codes require too specific explanations with no extra benefit. In this case, we know the simulation was purged along with isPremiumUser so we don't have more information. What we are logging is the act of purging, not the act of purging, because they are a non-premium user.

Reply:
Ok, thanks.